SUPPORTED_CLIENTS.md

```diff
@@ -16,7 +16,7 @@ These are the clients we actively test against to check are compatible with go-m
 - Java/JVM
   - [mariadb-java-client](#mariadb-java-client)
 - Go
-  - [go-mysql-driver/mysql](#go-mysql-driver-mysql)
+  - [go-mysql-driver/mysql](#go-sql-drivermysql)
```

Comment:
just to check, is this anchor correct?

Reply:
It is. It did not work before the change.